paper-input-behavior.html

```diff
@@ -384,6 +384,12 @@
         label.id = labelledBy;
       }
       this._ariaLabelledBy = labelledBy;
+    },
+
+    _onChange: function(event) {
+      this.fire('iron-change', {
```

Comment:
Ok. I did a whole bunch of testing and thinking and double-checking with @cdata and I think what we want to do is still fire `change`, but only in the shadow dom. So something like this:
```
_onChange:function(event) {
      // In the Shadow DOM, the `change` event is not leaked into the
      // ancestor tree, so we must do this manually.
      // See https://w3c.github.io/webcomponents/spec/shadow/#events-that-are-not-leaked-into-ancestor-trees.
      if (this.shadowRoot) {
        this.fire(event.type, {sourceEvent: event}, {
          node: this,
          bubbles: event.bubbles,
          cancelable: event.cancelable
        });
      }
    }
```

We can't add the `value` in the event, since we're trying to match the event that gets fired in the shady DOM.

WDYT?